client/src/App.js

```diff
@@ -1,24 +1,29 @@
 import React from 'react';
-import logo from './logo.svg';
+import {BrowserRouter, Route} from 'react-router-dom'
```

Comment:
Huh, I didn't expect this, but create-react-app works really well with react-router. I thought refreshing when at `http://localhost:3000/player` would return a 404, but it takes you to the right component. I'm used to just serving a directory, so that would never work (I have to use `HashRouter` instead of `BrowserRouter`). Real cool!